### PR TITLE
`Programming exercises`: Highlight feedback expanding button

### PR DESCRIPTION
**Pull request by daniels98it**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context

In some cases, it has been found that new users have difficulty in finding/using the button to show the full feedback, reporting that feedback messages aren't useful.

### Description

Adding an explicit 'see more' in bolder font should help new users finding/using this functionality.

#### Code Review
- [x] Review 1
- [ ] Review 2
#### Manual Tests
- [x] Test 1
- [ ] Test 2

#### Before

{5770C681-87D2-47C4-B4E8-C9617FB95B6D}

#### After

{C8064BBF-9FA4-4B09-AF9E-CCEFB50D9615}

### CODE REVIEW

**Review by FlorianGlombik — APPROVED** [review]
Tested on ts1, it is displayed as shown on the screenshot within the PR description. The code looks fine too!

File: src/main/webapp/i18n/en/result.json [hunk 1]
```diff
@@ -70,7 +70,8 @@
                     "tooltipOnlineIde": "No result was found / could be generated in time for your most recent submission. Please check your code manually as far as possible."
                 }
             },
-            "notLatestSubmission": "Note: This result and feedback is <strong>not</strong> based on the latest submission."
+            "notLatestSubmission": "Note: This result and feedback is <strong>not</strong> based on the latest submission.",
+            "seeMore": "See more"
```
Comment (b-fein) [inline src/main/webapp/i18n/en/result.json:74]:
```suggestion
            "seeMore": "Show more"
```
Intuitively ‘See more’ sounds a bit weird to me, but can’t really explain why. Maybe I see it more like ‘Show more’ as shorthand for ‘Click here to show more information’. That would also align better with the German variant.

**Review by b-fein — APPROVED** [review]
Code looks good, works as expected as well. :+1: Nice little UI improvement!

**Comment by krusche** [comment]
Could we also extend the preview? The feedback examples in your screenshot are really short.
I think the "Show more" button is here to avoid showing feedback with 20 lines. But all simple and short feedback should be displayed completely to avoid that students have to click on Show more all the time.

**Comment by daniels98it** [comment]
> Could we also extend the preview? The feedback examples in your screenshot are really short. I think the "Show more" button is here to avoid showing feedback with 20 lines. But all simple and short feedback should be displayed completely to avoid that students have to click on Show more all the time.

@krusche I see the "Show more" button main functionality to be the displaying of 'stacktraces' for the actual error, which can get quite big if expanded.. In the specific case of Python exercises for example, the feedback preview contains the actual error and the 'expanded' feedback contains a nice visualization about where the problem is located.. (see screenshot)
Screenshot (26)

**Review by cremertim — COMMENTED** [review]
tested on ts1, everything works as expected

**Review by cremertim — APPROVED** [review]

**Comment by krusche** [comment]
> > Could we also extend the preview? The feedback examples in your screenshot are really short. I think the "Show more" button is here to avoid showing feedback with 20 lines. But all simple and short feedback should be displayed completely to avoid that students have to click on Show more all the time.
> 
> @krusche I see the "Show more" button main functionality to be the displaying of 'stacktraces' for the actual error, which can get quite big if expanded.. In the specific case of Python exercises for example, the feedback preview contains the actual error and the 'expanded' feedback contains a nice visualization about where the problem is located.. (see screenshot) Screenshot (26)

In this case it makes sense, yes! I have to double check what happens in the Java template. When the feedback is only one line long, there should be no expansion needed

**Pull request merged**